tools/opscan/cmd/main.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/Layr-Labs/eigenda/api/grpc/node"
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/Layr-Labs/eigenda/disperser/dataapi"
+	"github.com/Layr-Labs/eigenda/disperser/dataapi/subgraph"
+	"github.com/Layr-Labs/eigenda/tools/opscan"
+	"github.com/Layr-Labs/eigenda/tools/opscan/flags"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/urfave/cli"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var (
+	version   = ""
+	gitCommit = ""
+	gitDate   = ""
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Version = fmt.Sprintf("%s,%s,%s", version, gitCommit, gitDate)
+	app.Name = "opscan"
+	app.Description = "operator network scanner"
+	app.Usage = ""
+	app.Flags = flags.Flags
+	app.Action = RunScan
+	if err := app.Run(os.Args); err != nil {
+		log.Fatal(err)
+	}
+}
+
```

Comment:
This can be made as a library, which is used to implement both a cmd tool, as well as an API at dataapi server to monitor the DA network.